tests/unit/database_model_generators/service.py

```diff
@@ -0,0 +1,22 @@
+"""
+
+"""
+from topchef.database.models.service import Service
+from hypothesis.strategies import composite
+from hypothesis.strategies import text, dictionaries
+
+
+@composite
+def services(
+        draw,
+        names=text(),
+        descriptions=text(),
+        registration=dictionaries(text(), text()),
+        results=dictionaries(text(), text())
+) -> Service:
```

Comment:
Also, figure out some way to give this pride of place in the model generators. Provide a primer for property-driven testing as well.